pydra/engine/submitter.py

```diff
@@ -13,6 +13,7 @@ class Submitter:
     # TODO: runnable in init or run
     def __init__(self, plugin):
         self.loop = get_open_loop()
+        self._running_loop = self.loop.is_running()
```

Comment:
Perhaps a more intuitive way of thinking about it:

```suggestion
        self._own_loop = not self.loop.is_running()
```